pkg/adapter/incoming.go

```diff
@@ -3,12 +3,23 @@ package adapter
 import (
 	"context"
 	"crypto/subtle"
+	"encoding/json"
 	"fmt"
+	"github.com/golang-jwt/jwt"
```

Comment:
please use the versioned version adn the same version as what we have on ghinstallaton

```jwt "github.com/golang-jwt/jwt/v4"```

Reply:
Updated